runtime/lua/vim/health.lua

```diff
@@ -327,17 +327,38 @@ end
 ---                            `:checkhealth vim.* nvim` will healthcheck `vim.lsp`, `vim.treesitter`
 ---                            and `nvim` modules.
 function M._check(mods, plugin_names)
+  local arg = ''
+  local t = vim.split(plugin_names, '%s')
```

Comment:
This is a simple implementation that does not change ex_checkhealth at the moment. This makes the meaning of the parameter a bit strange.  currently we don't have float modifier so ++flag is used. It might be better to modify ex_checkhelath and then add a new parameter here like`mods, plugin_names, arg`.  Is this better or add a float modifier?